### DISCUSSION TITLE
Timeline style scrollLineHourMode and showLineHourMode

### DISCUSSION THREAD

**Discussion by phuanggh** [kvyatkovskys/KVKCalendar#344] [Q&A]
Hi @kvyatkovskys, I saw these properties on `style.timeline`, but I didn't find the differences after applying them to the example project. What are the purpose of them?

**Comment by kvyatkovskys** [comment]
HI @phuanggh!
- `scrollLineHourMode` to scroll to specific hour we open a day or week
- `showLineHourMode` to show or hide the red horizontal line on timeline

**Reply by phuanggh** [reply]
Hi @kvyatkovskys , thanks for answering me. 
Could you tell me more about what each value of `scrollLineHourMode` means?
Thanks!

**Reply by kvyatkovskys** [reply]
- `always` scroll always when we open a day
- `today` scroll today only
- `forDate(Date)` scroll for a specific date only
- `never` scroll is disabled
- `onlyOnInitForDate(Date)` scroll only first time when we open calendar

**Reply by phuanggh** [reply]
> `scrollLineHourMode` to scroll to specific hour we open a day or week
Hi @kvyatkovskys , how should we set the specific hour we want it to scroll to?

**Reply by kvyatkovskys** [reply]
You need to create a date with the specific hour 
```swift
var dateComponents = DateComponents()
dateComponents.hour = 9
let date = Calendar.current.date(from: dateComponents)
scrollLineHourMode = .forDate(date)
```
